cypress/integration/teams.spec.js

```diff
@@ -0,0 +1,26 @@
+describe("Teams", () => {
+    beforeEach(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+        cy.visit("/teams");
+    });
+
+    it("lists all teams", () => {
+        cy.get(".govuk-table__row").should("have.length", 2);
+
+        const expected = ["Cool Team", "Supervision — Very Cool", "1"];
+
+        cy.get(".govuk-table__body > .govuk-table__row")
+            .children()
+            .each(($el, index) => {
+                cy.wrap($el).should("contain", expected[index]);
+            });
+    });
+
+    it("allows me to search for a team", () => {
+        cy.get("#f-search").clear().type("no such team");
+        cy.get("button[type=submit]").click();
+
+        cy.get(".govuk-table__row").should("have.length", 1);
```

Comment:
I know this is just because of pact-mock, but it's confusing that "no such team" would return a single result. Could we change the search to "cool team" so it looks clearer?

Reply:
I think the other confusing thing is the selector I used picks up the header row, hence 1 when there are no results. I'll change to do both.